zenodo/modules/deposit/indexer.py

```diff
@@ -70,3 +70,8 @@ def indexer_receiver(sender, json=None, record=None, index=None,
     else:
         json['_updated'] = record.updated
     json['_created'] = record.created
+
+    # Compute filecount and total file size
+    files = json.get('_files', [])
+    json['filecount'] = len(files)
+    json['filesize'] = sum([f.get('size', 0) for f in files])
```

Comment:
I agree that it is kind of misleading, the reasoning behind the name was to ensure that all the file related fields are predictable and simple.... ``filename, filetype, filesize, filecount``.

I'd prefer either ``size`` or ``bytesize``. WDYT?